kymatio/scattering1d/core/scattering1d.py

```diff
@@ -92,7 +96,8 @@ def scattering1d(x, pad, unpad, backend, J, psi1, psi2, phi, pad_left=0,
         # Convolution + downsampling
         j1 = psi1[n1]['j']
 
-        k1 = max(j1 - oversampling, 0)
+        sub1_adj = min(j1, log2_T) if average else j1
```

Comment:
what does `sub1_adj` mean?

Reply:
"subsampling 1 adjusted", didn't think much on this